packages/babel-plugin-emotion/src/macro.js

```diff
@@ -30,7 +30,7 @@ export let createEmotionMacro = (instancePath: string) =>
               babel,
               state,
               path,
-              shouldLabel: state.opts.autoLabel
+              shouldLabel: true
```

Comment:
shouldnt this be dependant on `process.env.NODE_ENV` ("runtime" one) or at least configurable with the default being true?

Reply:
It's dependent on the build time `process.env.NODE_ENV`, the check is [here](https://github.com/emotion-js/emotion/blob/6cb8d15438b01b8623af42f304d5ea3032332187/packages/babel-plugin-emotion/src/utils/label.js#L28-L30). Since the main use case for the macros will be Create React App and `process.env.NODE_ENV` is set to production for production builds in Create React App, I think doing this is safe.

Reply:
Yeah, the reason it's like that is that the styled transform doesn't insert the label into the styles, it inserts it into the the styled call so shouldLabel should be disabled there.